x-pack/platform/plugins/shared/lens/public/visualizations/metric/toolbar/appearance_popover.tsx

```diff
@@ -0,0 +1,395 @@
+): MetricLayoutWithDefault => {
+  let config = { ...METRIC_LAYOUT_BY_POSITION[primaryMetricPosition] };
+
+  if (!hasMetricIcon) {
+    const { iconAlign, ...rest } = config;
+    config = rest;
+  }
+
+  if (!hasSecondaryMetric) {
+    const { secondaryAlign, ...rest } = config;
+    config = rest;
+  }
+
+  return config;
+};
+
+interface AppearancePopoverProps {
+  state: MetricVisualizationState;
+  setState: (newState: MetricVisualizationState) => void;
+  groupPosition?: ToolbarPopoverProps['groupPosition'];
+}
+
+export function AppearancePopover({ state, setState, groupPosition }: AppearancePopoverProps) {
+  const hasSecondaryMetric = !!state.secondaryMetricAccessor;
+  const hasMetricIcon = hasIcon(state.icon);
+
+  const disabledStates = {
+    subtitle: !!state.breakdownByAccessor,
+    secondaryAlign: !hasSecondaryMetric,
+    iconAlign: !hasMetricIcon,
+  };
+
+  return (
+    <ToolbarPopover
+      title={i18n.translate('xpack.lens.metric.appearancePopover.title', {
+        defaultMessage: 'Appearance',
+      })}
+      type="titlesAndText"
+      groupPosition={groupPosition}
+      buttonDataTestSubj="lnsTextOptionsButton"
+    >
+      <AppearanceOptionGroup
+        title={i18n.translate('xpack.lens.metric.appearancePopover.titleAndSubtitle.title', {
+          defaultMessage: 'Title and subtitle',
+        })}
+      >
+        <SubtitleOption
+          value={state.subtitle}
+          onChange={(subtitle) => {
+            setState({
+              ...state,
+              subtitle,
+            });
+          }}
+          isDisabled={disabledStates.subtitle}
+        />
+        <AppearanceOption
+          label={i18n.translate('xpack.lens.metric.appearancePopover.alignment', {
+            defaultMessage: 'Alignment',
+          })}
+          value={state.titlesTextAlign ?? metricStateDefaults.titlesTextAlign}
+          options={alignmentOptions}
+          onChange={(id) => {
+            setState({
+              ...state,
+              titlesTextAlign: id,
+            });
+          }}
+          isIconOnly
+          dataTestSubj="lens-metric-appearance-title-and-subtitle-alignment-btn"
+        />
+        <AppearanceOption
+          label={i18n.translate('xpack.lens.metric.appearancePopover.fontWeight', {
+            defaultMessage: 'Font weight',
+          })}
+          value={state.titleWeight ?? metricStateDefaults.titleWeight}
+          options={fontWeightOptions}
+          onChange={(id) => {
+            setState({
+              ...state,
+              titleWeight: id,
+            });
+          }}
+          dataTestSubj="lens-metric-appearance-title-and-subtitle-font-weight-btn"
+        />
+      </AppearanceOptionGroup>
+      <EuiHorizontalRule margin="xs" />
+      <AppearanceOptionGroup
+        title={i18n.translate('xpack.lens.metric.appearancePopover.primaryMetric.title', {
+          defaultMessage: 'Primary metric',
+        })}
+      >
```

Comment:
Do you mind moving this to the top as the first group in the popover?
I believe that, since the position button controls many other things, it should be first in the visual hierarchy.
You don't want to start clicking things and suddently one button changes it all :D
@gvnmagni  do you agree?
The other option was to add a dropdown instead where you can select the layouts (legacy, new, custom). as soon as something is different from the selected layout it become a custom layout.
@gvnmagni can you validate and incase confirm if we want to go in this direction?